fastlane/spec/actions_specs/setup_circle_ci_spec.rb

```diff
@@ -42,7 +42,7 @@ def check_keychain_created
         check_keychain_nil
       end
 
-      it "works on MacOS Environment when forced" do
+      it "works on macOS Environment when forced" do
```

Comment:
The brand should generally be written like this (except when it's for certain directory paths or classes...)